three-tier-main.tf

```diff
@@ -277,7 +279,7 @@ resource "aws_lb_target_group_attachment" "external-elb2" {
 resource "aws_lb_listener" "external-elb" {
   load_balancer_arn = aws_lb.external-elb.arn
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure one HTTPS listener is configured for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one HTTPS listener could impact the confidentiality of data in transit.
 
**Remediation**
Elastic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: tenablecs-